### ISSUE TITLE
A2Desktop hangs on selector when starting

### ISSUE THREAD

**Issue by suaide** [a2stuff/a2d#485]
Tried most recent A2Desk (1.2.alpha31 and alpha30) on a TK3000 (apple IIe clone) with fastchip. After loading it will hang on selector. Mouse does not move and keyboard does not work. I tried with version 1.1 and works without problems

**Comment by inexorabletash** [comment]
I am unable to repro using MAME (via [Ample](https://github.com/ksherlock/ample)):

`mame64 tk3000 -skip_gameinfo -window -nomax -sl4 mouse -sl7 cffa2 -hard1 '.../HD32MB.2mg' -nothrottle`

...where I've got that HD image set up to launch `Selector`. 

To eliminate some possibilities:
* Are you able to try it without the Fastchip card? 
* If you remove Selector so it launches directly into DeskTop, does DeskTop start up correctly?

**Comment by inexorabletash** [comment]
PS: @suaide - if you're using the TK3000, any interest in helping localize to Portuguese ?

**Comment by inexorabletash** [comment]
I was able to repro, and have a hunch as to what going on: Selector doesn't handle errors when loading the `SELECTOR.LIST` file, and I think it's loaded into uninitialized memory. The first two bytes are the numbers of entries; if those are garbage, random reads and bad calls are going to happen. 

And this matches what I saw if the file is deleted. Sometimes Selector loads and hangs, sometimes crazy crashes, and sometimes it jumps right into DeskTop. 

Fix should be easy.
